components/feature/pwa/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<resources>
+
+    <style name="Theme.AppCompat.Translucent" parent="Theme.AppCompat.NoActionBar">
```

Comment:
Since you do not have any UI here you could also just extend from `Activity` (instead of using AppCompat) and use the system's translucent theme. A little bit less code.

Reply:
I didn't use the system theme because doesn't work with `AppCompatActivity` and we may want to use it as a `LifecycleOwner` in the future.